be/src/olap/wal/wal_table.cpp

```diff
@@ -45,10 +45,6 @@ WalTable::WalTable(ExecEnv* exec_env, int64_t db_id, int64_t table_id)
 }
 WalTable::~WalTable() {}
 
-#ifdef BE_TEST
-Status k_stream_load_exec_status;
```

Comment:
k_stream_load_exec_status in wal_manager_test.cpp need to be removed as well, now the be ut complie is failed